content/development/coding-guidelines.md

```diff
@@ -348,8 +348,48 @@ const uint32 kOpenFile = 'open';
 ```
 
 *   In some cases you might have to extend the copyright list of an existing file. Instead of using the alternative scheme like in the latter example the **preferred** method is to add a copyright line for "Haiku, Inc." and then list authors like in the first example.
-*   In header files, there is no blank line between the license text and the header guard.
-*   After the copyright header (including the header guard in header files), there are exactly two blank lines before the rest contents.
+*   In header files, there is no blank line between the license text and the header guard but following that should be two blank lines. There should also be two blank lines before the close of the header guard as shown below;
+
+```cpp
+/*
+ * Copyright 2004-2007 Haiku, Inc. All rights reserved.
+ * Distributed under the terms of the MIT License.
+ *
+ * Authors:
```

Comment:
This block is optional, really. I think in newer files it's started to fall by the wayside. But I guess we can leave it in these guidelines...